setup.py

```diff
@@ -23,27 +23,6 @@
 }
 
 
-class Cmd(install):
-    """Custom clean command to tidy up the project root."""
-
-    def initialize_options(self):
-        install.initialize_options(self)
-
-    def finalize_options(self):
-        install.finalize_options(self)
-
-    def run(self):
-        install.run(self)
-        dirs = [
-            'pyam.egg-info',
-            'build',
-        ]
-        for d in dirs:
-            if os.path.exists(d):
-                print('removing {}'.format(d))
-                shutil.rmtree(d)
-
-
 # thank you https://stormpath.com/blog/building-simple-cli-interfaces-in-python
 class RunTests(Command):
     """Run all tests."""
```

Comment:
Is this class redundant now that you're using `"cmdclass": versioneer.get_cmdclass(),` below? Looking at https://stormpath.com/blog/building-simple-cli-interfaces-in-python again, I think it needs to be setup something like https://github.com/znicholls/netcdf-scm/blob/master/setup.py for this to actually do anything.

Reply:
Yes, it is. I wasn't sure what exactly to do with it, since I didn't add it (I believe you did, right?). 

In any case, I can mirror your netcdf-scm style or we can just remove it. I always run `pytest tests` and not `python setup.py test`. Do you have a preference?